bluepyopt/parameters.py

```diff
@@ -33,6 +34,9 @@ def __init__(self, name, value=None, frozen=False, bounds=None):
         self._value = value
         self.check_bounds()
         self.frozen = frozen
+        self.param_dependancies = param_dependancies
```

Comment:
spelling error

Reply:
> So it seems LFPy requires mpi4py? Installation seems to fail during the tests. Can we disable mpi4py in LFPy, since I assume we don't need it?

Yes, it's only used in the network.py module of LFPy, and we do not use that one.

Reply:
fixed